tests/pass/shims/x86/intrinsics-x86-bmi.rs

```diff
@@ -78,8 +95,12 @@ unsafe fn test_bmi_32() {
         let s = 0b0001_0010u32;
         assert_eq!(_bzhi_u32(n, 5), s);
 
-        for i in 0..=32 {
-            assert_eq!(_bzhi_u32(u32::MAX, i), u32::MAX.checked_shr(32 - i).unwrap_or(0),);
+        // Ensure that indices larger than the bit count are covered.
+        for i in 0..1000 {
```

Comment:
How long does this test take now? 1000 loop iterations can become somewhat slow in Miri...

Reply:
On my system, the execution was pretty much instant.

Reply:
True, this is faster than I expected...
Still, should be enough to cover `0..=512`. That also looks less arbitrary than 1000. ;)